File: NavitiaSDKSandbox/NavitiaSDKSandbox/NavitiaEndpoints.swift
//
// Created by Chakkra CHAK on 25/04/2017.
// Copyright (c) 2017 Chakkra CHAK. All rights reserved.
//

import Foundation

open class NavitiaEndpoints {
    let navitiaConfiguration:NavitiaConfiguration
    // <Generated>
    open let places: EndpointPlaces
    // </ Generated>

    public init(navitiaConfiguration:NavitiaConfiguration) {
        self.navitiaConfiguration = navitiaConfiguration
        // <Generated>
        self.places = EndpointPlaces(navitiaConfiguration: self.navitiaConfiguration)
        // </ Generated>
    }
}

File: android/NavitiaSDKSandbox/src/main/java/org/kisio/labs/NavitiaSDKSandbox/NavitiaEndpoints.java
package org.kisio.labs.NavitiaSDKSandbox;

import org.kisio.labs.NavitiaSDKSandbox.Places.EndpointPlaces;

/**
 * Created by cchak on 27/04/2017.
 */
public class NavitiaEndpoints {
    private NavitiaConfiguration navitiaConfiguration;
    private EndpointPlaces places;

    public NavitiaConfiguration getNavitiaConfiguration() {
        return navitiaConfiguration;
    }

    public EndpointPlaces getPlaces() {
        return places;
    }

    public NavitiaEndpoints(NavitiaConfiguration navitiaConfiguration) {
        this.navitiaConfiguration = navitiaConfiguration;
        this.places = new EndpointPlaces();
    }
}
